.isort.cfg

```
@@ -0,0 +1,11 @@
+[settings]
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+combine_as_imports=True
+use_parentheses=True
+line_length=126
```

```suggestion
line_length=120
```
Everything else is 120 chars, and `github` diffview is max `125` chars.

---

Yeah good catch 👍